ui/store/institutional/institution-background.test.js

```diff
@@ -97,4 +123,52 @@ describe('Institution Actions', () => {
       expect(setWaitForConfirmDeepLinkDialog).toBeDefined();
     });
   });
+
+  describe('#showInteractiveReplacementTokenBanner', () => {
+    it('should test showInteractiveReplacementTokenBanner action', async () => {
+      const dispatch = jest.fn();
+
+      await showInteractiveReplacementTokenBanner({
+        url: 'testUrl',
+        oldRefreshToken: 'testToken',
+      })(dispatch);
+
+      expect(submitRequestToBackground).toHaveBeenCalledWith(
+        'showInteractiveReplacementTokenBanner',
+        [{ url: 'testUrl', oldRefreshToken: 'testToken' }],
+      );
+    });
+  });
+
+  describe('#setTypedMessageInProgress', () => {
+    it('should test setTypedMessageInProgress action', async () => {
+      const dispatch = jest.fn();
+
+      await setTypedMessageInProgress('testMsgId')(dispatch);
+
+      expect(showLoadingIndication).toHaveBeenCalled();
+      expect(submitRequestToBackground).toHaveBeenCalledWith(
+        'setTypedMessageInProgress',
+        ['testMsgId'],
+      );
+      expect(forceUpdateMetamaskState).toHaveBeenCalledWith(dispatch);
+      expect(hideLoadingIndication).toHaveBeenCalled();
+    });
+  });
+
+  describe('#setPersonalMessageInProgress', () => {
```

Comment:
would be great to have error handling tests as well ;)

Reply:
Hey @DDDDDanica, at the moment we are not going to test the error parts